core/templates/dev/head/pages/library/library.html

```diff
@@ -204,6 +230,23 @@ <h2 ng-class="{'active': activeGroupIndex === $index}" class="oppia-library-grou
     </div>
   </div>
 
+  <script type="text/ng-template" id="modals/removeActivity">
```

Comment:
Pull this to a separate file so it can be retrieved asynchronously as needed. See example [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js#L373) and [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/exploration_editor/welcome_modal_directive.html) for the new way of doing this.